.github/pull_request_template.md

```diff
@@ -35,39 +35,40 @@ Read the following guidelines and remove them before opening the pull request.
   to fix bugs (by changing code only to make it nicer you might introduce a bug).
 * Set the optimal number of cores that the tool can use (check if parallelization
   can be performed).
-* Description of tool should be self explanatory with links to tool's homepage
-  and publication.
+* Description of the tool should be self explanatory with links to the tool's
+  homepage and publication.
 * Write explanatory input labels and add short command line parameter names
   in square brackets (for example: Maximum length [``--maxlength``]).
 * Where possible provide default values for the process input fields.
-  If they are provided, then ``required: false`` is not needed.
+  If they are provided, then ``required = false`` is not needed.
 * Add descriptive error and warning messages.
-* Place a ``re-checkrc`` call after each process step (call of a tool, a
-  Bash command, a Python script or an R script).
-* Add ``re-progress`` calls where appropriate.
+* Use ``Cmd()`` for each call in the process (call of a tool, a
+  Bash command, a Python script or an R script) and handle return codes.
+* Add ``self.progress()`` calls where appropriate.
 
 ### Commit and PR messages ###
 
 * Each commit should be minimal (i.e. 1 change) and self-contained (including
   tests).
 * Changes to Docker images need to be accepted in a separate pull request.
   Before the new images can be used for automated testing of your PR, they
-  must be manually tagged and pushed to Docker Hub.
+  must be manually tagged and pushed to Amazon ECR.
 * Mark incomplete PRs with the [WIP] (Work In Progress) tag.
 * Do not paste links to private repositories from the public one.
 
 ### Tests ###
 
-* Add meaningful names of test files with spaces:
-  * Genomes: ``genome species.fa.gz``
-  * Annotations: ``annotation species.gtf.gz``
+* Add meaningful names of test files with underscores:
```

Comment:
Maybe this comment is not so relevant. In yaml processes, we had to be very careful with spaces in filenames, but in Python processes, this is not an issue. Maybe just "Add meaningful names of test files" would suffice.

Reply:
That is a good point, I will change it to "Add meaningful names of test files". But I would keep the underscores in example file names as it is still a better practice to use underscores instead of spaces.